### PR TITLE
Frontend structure for Other contacts in detailed view

### PR DESCRIPTION
**Pull request by oliveiraigorm**
Contacts have to be separated by commas. Regex for matching the phone number can be developed more to allow spaces or dashes 

image

image

### CODE REVIEW

File: src/components/ClientModal.vue [hunk 1]
```diff
@@ -326,6 +359,10 @@ p span {
 	resize: none;
 }
 
+li::before {
+	content: '•';
```
Comment (oliveiraigorm) [inline src/components/ClientModal.vue:363]: @mwalbeck  I'm having a lint issue here, when I run "make lint-fix" it changes it to "•", but then it gives me this error:  "Expected single quotes  string-quotes" and if I change it to '•' I also get a lint error. Can you add some rule to eslint to fix that please?

Comment (mwalbeck) [reply src/components/ClientModal.vue]: Done!

**Comment by mwalbeck** [comment]
Handling of an empty Other contacts field is needed as the modal won't load at all when the field is empty.

File: src/components/ClientModal.vue [hunk 2]
```diff
@@ -186,6 +208,11 @@ export default {
 		displayTimezone() {
 			return this.mutableClient.timezone.slice(0).replace("_", " ");
 		},
+		contactsList() {
+			return this.mutableClient.contacts
+				? this.mutableClient.contacts.split(",")
+				: "";
+		},
```
Comment (oliveiraigorm) [inline src/components/ClientModal.vue:215]: @mwalbeck I've added this validation that fixes the issue with the modal not opening.

File: src/components/ClientModal.vue [hunk 3]
```diff
@@ -105,16 +110,33 @@
 						</p>
 					</div>
 					<div class="col ml-22">
-						<h3>Other Contacts</h3>
+						<h3 v-if="mutableClient.contacts || editMode">
+							Other Contacts
+						</h3>
```
Comment (oliveiraigorm) [inline src/components/ClientModal.vue:115]: @mwalbeck I've also added this option not to render the other contacts column if it was empty. That wasn't decided in the UI but if you think is a nice feature we could keep it and consider doing it also to the description.

Comment (mwalbeck) [reply src/components/ClientModal.vue:115]: Let's keep it and bring it up on Monday

**Review by mwalbeck — APPROVED** [review]

**Pull request merged**